framework/src/bundle/ModuleMachOFile.cpp

```diff
@@ -19,7 +19,7 @@
 
 =============================================================================*/
 
-#include "ModuleObjFile_p.h"
+#include "ModuleObjFile.h"
```

Comment:
I think we should replace "Module" with "Bundle" for these file names. The classes inside these files have "Bundle" as prefix.